### ISSUE TITLE
mqbench导出swintransformer时候，导出onnx 发现一些matmul节点没有插上量化节点

### ISSUE THREAD

**Issue by HawkL327** [ModelTC/MQBench#153]
![image](https://user-images.githubusercontent.com/33046518/181162438-a62cd5ff-9c9f-4ce1-b8c7-db7d05453ef0.png)

**Comment by Tracin** [comment]
backend_type是?

**Comment by HawkL327** [comment]
> backend_type是?

tensorrt

**Comment by Tracin** [comment]
> > backend_type是?
> 
> tensorrt

可以使用Tensorrt_NLP试试

**Comment by HawkL327** [comment]
> > > backend_type是?
> > 
> > 
> > tensorrt
> 
> 可以使用Tensorrt_NLP试试

试了，还是没有

**Comment by Tracin** [comment]
可以提供下模型定义吗

**Comment by HawkL327** [comment]
源码是
attn = (F.normalize(q, dim=-1) @ F.normalize(k, dim=-1).transpose(-2, -1)) 
这样会转失败，于是改成了
attn = torch.matmul(F.normalize(q, dim=-1),F.normalize(k, dim=-1).transpose(-2, -1))
好像就是这个matmul没办法插入节点，有没有什么办法绕过？

**Comment by HawkL327** [comment]
[Swin-Transformer.tar.gz](https://github.com/ModelTC/MQBench/files/9196434/Swin-Transformer.tar.gz)
git diff可以看到修改部分

**Comment by HawkL327** [comment]
我发现这个网络的matmul如果全设int8的话，精度可能会有问题。有办法可以指定设置吗

**Comment by HawkL327** [comment]
有一处是 qkv = F.linear(input=x, weight=self.qkv.weight, bias=qkv_bias) ，改成nn.Linear就行了
